Packaging.props

```diff
@@ -2,14 +2,14 @@
   <PropertyGroup>
     <PreReleaseLabel>preview2</PreReleaseLabel>
     <PackageDescriptionFile>$(ProjectDir)pkg/descriptions.json</PackageDescriptionFile>
-    <PackageLicenseFile>$(ProjectDir)LICENSE</PackageLicenseFile>
-    <PackageThirdPartyNoticesFile>$(ProjectDir)pkg/ThirdPartyNotices.txt</PackageThirdPartyNoticesFile>
+    <PackageLicenseFile>$(ProjectDir)LICENSE.TXT</PackageLicenseFile>
```

Comment:
Yes, first for consistency with other repos. second, it is requested by @richlander so it'll be easy to open the file on Windows